app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -83,7 +85,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
         if (Utils.isOnline(getActivity())) {
             setIsLoading(true);
             App.getInstance().getModelCache()
-                .getAsync(Const.CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
+                .getAsync(CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
```

Comment:
That commit has reverted and then I moved all cache related consts to ModelCache class. Can you take a look?